apps/api/src/routers/admin.py

```diff
@@ -149,28 +152,49 @@ async def applicant_summary() -> dict[ApplicantStatus, int]:
 
 @router.post("/review")
 async def submit_review(
-    applicant: str = Body(),
-    decision: Decision = Body(),
+    applicant_review: ReviewRequestModel,
     reviewer: User = Depends(require_role({Role.REVIEWER})),
 ) -> None:
-    """Submit a review decision from the reviewer for the given applicant."""
-    log.info("%s reviewed applicant %s", reviewer, applicant)
+    """Submit a review decision from the reviewer for the given hacker applicant."""
+    log.info("%s reviewed hacker %s", reviewer, applicant_review.applicant)
 
-    review: OtherReview = (utc_now(), reviewer.uid, decision)
+    review: Review = (utc_now(), reviewer.uid, applicant_review.score)
 
```

Comment:
Suggestion: even though the frontend hides the ability to add reviews once there have been 2 reviewers, it's still important to double check the number of unique reviewers in the backend before attempting to add a review to the DB. In practice, since only Hack orgs have access to the admin portal and its API endpoints, this shouldn't matter too much but for the sake of robustness and extra security in the case of a malicious user trying to submit a review straight to this endpoint.

Reply:
A reviewer will be allowed to change their score afterwards. To be more clear on what I meant previously by double checking the number of unique reviewers, if the user isn't already one of the 2 reviewers, then prevent a new review from being added, but if the user is one of the existing reviewers, then allow them to add another (updated) review.

Reply:
Follow up to this, it seems that log/applications will be deciding how to assign apps to reviewers in their next meeting in a few days.